packages/rspack/src/builtin-plugin/LightningCssMiminizerRspackPlugin.ts

```diff
@@ -1,26 +1,72 @@
 import {
 	BuiltinPluginName,
+	type RawLightningCssBrowsers,
 	type RawLightningCssMinimizerRspackPluginOptions
 } from "@rspack/binding";
 
+import browserslist from "browserslist";
+import {
+	type Drafts,
+	type FeatureOptions,
+	type NonStandard,
+	type PseudoClasses,
+	type Targets,
+	browserslistToTargets,
+	toFeatures
+} from "../builtin-loader/lightningcss";
+import type { AssetConditions } from "../util/assetCondition";
 import { create } from "./base";
 
-export type LightningCssMinimizerRspackPluginOptions =
-	Partial<RawLightningCssMinimizerRspackPluginOptions>;
+export type LightningCssMinimizerRspackPluginOptions = {
+	test?: AssetConditions;
+	include?: AssetConditions;
+	exclude?: AssetConditions;
+	removeUnusedLocalIdents?: boolean;
+	minimizerOptions?: {
+		errorRecovery?: boolean;
+		targets?: Targets | string[] | string;
+		include?: FeatureOptions;
+		exclude?: FeatureOptions;
+		draft?: Drafts;
+		nonStandard?: NonStandard;
+		pseudoClasses?: PseudoClasses;
+		unusedSymbols?: string[];
+	};
+};
 
 export const LightningCssMinimizerRspackPlugin = create(
 	BuiltinPluginName.LightningCssMinimizerRspackPlugin,
 	(
 		options?: LightningCssMinimizerRspackPluginOptions
 	): RawLightningCssMinimizerRspackPluginOptions => {
+		const { include, exclude, draft, nonStandard, pseudoClasses } =
+			options?.minimizerOptions ?? {};
+		const targets = options?.minimizerOptions?.targets ?? "Chrome 60"; // last not support es module chrome version
 		return {
-			errorRecovery: options?.errorRecovery ?? true,
-			unusedSymbols: options?.unusedSymbols ?? [],
-			removeUnusedLocalIdents: options?.removeUnusedLocalIdents ?? true,
-			browserslist: options?.browserslist ?? ["defaults"],
 			test: options?.test,
 			include: options?.include,
-			exclude: options?.exclude
+			exclude: options?.exclude,
+			removeUnusedLocalIdents: options?.removeUnusedLocalIdents ?? true,
+			minimizerOptions: {
+				errorRecovery: options?.minimizerOptions?.errorRecovery ?? true,
+				unusedSymbols: options?.minimizerOptions?.unusedSymbols ?? [],
+				include: include ? toFeatures(include) : undefined,
+				exclude: exclude ? toFeatures(exclude) : 1048575, // exclude all features, avoid downgrade css syntax when minimize
```

Comment:
1048575 is a magic number, can we explain how to calc this number in the comments?